src/Google.Api.Gax/CallTiming.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    public enum CallTimingType
+    {
+        /// <summary>
+        /// Call uses retry, specified with a <see cref="RetrySettings"/>.
+        /// </summary>
+        Retry,
+
+        /// <summary>
+        /// Call does not use retry; call expiration is specified with an <see cref="Google.Api.Gax.Expiration"/> .
+        /// </summary>
+        Expiration,
+    }
+
+    public sealed class CallTiming
```

Comment:
True... they also expand what is considered a failure, beyond the failures explicitly returned by the service (meaning the timeout case). 

The thing about the name "CallTiming" is that it doesn't capture the fact that failures are being handled at all. 

I've tried to find an overarching term that captures retry and timeout, but I don't know that there is a satisfying one. Maybe it just needs to be called "TimeoutAndRetrySettings." (I'm looking at this from the cross-language perspective; I think that the current Java model needs to be readjusted.)